samples/latest/HelloMvc/NuGet.Config

```diff
@@ -2,7 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/api/v2" />
+    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v2" />
```

Comment:
What do you guys think of switching to `aspnetcidev` for the samples in `latest`?